Team 1 - Project Proposal

brycecorbitt opened the pull request.

pooja-a-patel committed:
Update README.md

brycecorbitt committed:
Created proposal for first PR

charlieroberts commented:
Nice idea! I encourage you to really have some fun exploring what `<canvas>` can do. Not sure what the limits of react-canvas-draw are, but you could really make an interesting tool using the API that canvas provides. Having layers that you could potentially rotate / scale / translate might open up a lot of possibilities.

brycecorbitt committed:
Transferred code from a4

brycecorbitt committed:
Updated authentication and added Drawing API endpoints

brycecorbitt committed:
Added development page for Drawing Canvas

brycecorbitt committed:
Completed API Client

brycecorbitt committed:
Removed auth enforcement for gallery API call

brycecorbitt committed:
Fixed bug with API Login callback

brycecorbitt committed:
Generalized content feed, added feeds for drawings

brycecorbitt committed:
Updated navbar and routes

brycecorbitt committed:
Removed form from home screen

Commit:
react-canvas-draw updated

brycecorbitt committed:
Added react-color dependency

Commit:
Update draw.js

brycecorbitt committed:
Improved draw, added delete and edit to feeds

brycecorbitt committed:
eslint formatting

brycecorbitt committed:
Added carousel API endpoint and client method

brycecorbitt committed:
Small fix with color picker

brycecorbitt committed:
Implemented carousel

brycecorbitt committed:
Prepared client for build

brycecorbitt committed:
Added webpack build to server

brycecorbitt committed:
Please work

brycecorbitt committed:
I'm not asking anymore. Work!

pooja-a-patel committed:
Create README.md

brycecorbitt committed:
Final Commit

This was a very fun and enriching experience that will help us better our implementation for MQP!
